apps/daffio/src/index.html

```diff
@@ -7,9 +7,13 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="icon" type="image/x-icon" href="assets/appicons/favicon.ico">
+  <link rel="stylesheet" href="https://use.typekit.net/kqu7meg.css">
```

Comment:
Not needed

Reply:
resolved in 078b7c737e13f76433aa6d07a37d7603f0c49bb3